server/internal/infrastructure/wallet/liquid-standalone/service.go

```diff
@@ -168,6 +173,35 @@ func (s *service) VerifyMessageSignature(ctx context.Context, message, signature
 	return false, errors.New("not implemented")
 }
 
+func (s *service) GetCurrentBlockTime(ctx context.Context) (*ports.BlockTimestamp, error) {
+	tipURL, err := url.JoinPath(s.esploraURL, "blocks/tip/height")
```

Comment:
call `block/tips/hash` to get the hash of the latest block, then call `block/:hash` to get the object that includes height and timestamp (counts for block timestamp)